test/distributed/elastic/rendezvous/c10d_rendezvous_backend_test.py

```diff
@@ -36,9 +39,34 @@ def setUp(self) -> None:
     def _corrupt_state(self) -> None:
         self._store.set("torch.rendezvous.dummy_run_id", "non_base64")
 
+class FileStoreBackendTest(TestCase, RendezvousBackendTestMixin):
+    _store: ClassVar[FileStore]
+
+    @classmethod
+    def setUpClass(cls):
+        temp_dir = tempfile.mkdtemp(prefix="torch_rdzv_")
+        cls._temp_dir = temp_dir
+
+    @classmethod
+    def tearDownClass(cls):
+        shutil.rmtree(cls._temp_dir)
+
+    def setUp(self) -> None:
+        # Currently, filestore doesn't implement a delete_key method, so a new
+        # filestore has to be initialized for every test in order to have a
+        # clean slate.
+        _, path = tempfile.mkstemp(dir=self._temp_dir, suffix="dummy_run_id")
+
+        self._store = FileStore(path, 1)
+        self._backend = C10dRendezvousBackend(self._store, "dummy_run_id")
+
+    def _corrupt_state(self) -> None:
+        self._store.set("torch.rendezvous.dummy_run_id", "non_base64")
+
 
 class CreateBackendTest(TestCase):
     def setUp(self) -> None:
+        # These are the default parameters, which are tcp
```

Comment:
Since the default is to run `CreateBackendTest` with TCP, instead of adding separate test cases only for FileStore in this class and having multiple conditionals to separate the two, can we instead use inheritance? E.g.

```python
class CreateBackendTest(TestCase):
    # as before...

class CreateBackendTestFileStore(CreateBackendTest):
   def setUp(self):
       # override setUp method by setting self.params to be your filestore params

   def test_bad_min_and_max_nodes_filestore(self):
       # your implementation (I believe this was the only unique test for filestore that was added)
```
This would run the same common suite of tests for TCP and FileStore, as well as the additionally added one for FileStore.

Alternatively, you could split it up into a common class such as `CreateBackendTestCommon` and have `CreateBackendTestTCP` and `CreateBackendTestFileStore` both inherit from it, but that may be overkill.